### DISCUSSION TITLE
Can we download a specific collection?

### DISCUSSION THREAD

**Discussion by cancri-cheli** [Avnsx/fansly#97] [Q&A] [answered]
The scraper is configured to download a whole user by default, but some download options seem to have a collection option, so; how can we configure it just to download one of our collections?

**Comment by Avnsx** [comment, marked as answer]
No clue what "one of our collections" means. When you set ``download_mode`` to ``Collection`` it will just download everything in your collection.
https://github.com/Avnsx/fansly-downloader/blob/046961917bf2e5f2e3e0896176160e54699b9840/config.ini#L9
